css/src/components/form/radio.scss

```diff
@@ -23,7 +23,7 @@ $radio-spacing: 0.5em !default;
 
 @mixin radio-dot-checked {
 	background: $radio-dot-checked-color;
-	box-shadow: inset 0 0 0 0.25em $body-background;
+	box-shadow: inset 0 0 0 0.15em $body-background;
```

Comment:
It looks like hovered check on line 97 should be updated as well

![image](https://github.com/user-attachments/assets/8c246c7b-18ad-48a1-98d3-5d21db550db0)

Reply:
updated!

Reply:
Should we make a new variable for the 0.15em so it can be shared with the hovered check?

Reply:
pushed